src/main.ts

```diff
@@ -34,8 +34,33 @@ Vue.config.productionTip = false;
 Vue.use(VueCompositionApi);
 Vue.use(VueVirtualScroller);
 
+// see https://advancedweb.hu/how-to-add-timeout-to-a-promise-in-javascript/
+const timeout = (prom : any, time : any, exception : any) => {
+    let timer : any;
+    return Promise.race([
+        prom,
+        new Promise((_r, rej) => timer = setTimeout(rej, time, exception)),
+    ]).finally(() => clearTimeout(timer));
+};
+const timeoutError = Symbol('timeoutError');
+
 async function start() {
-    await initStorage(); // Must be awaited before starting Vue
+    // LocalStorage must be initalised before starting Vue
+    // FIXME: Remove this as soon the Apple fixes the Safari bug
+    try {
+        // Due to a bug in iOS 14.6 IndexedDB initalisation *always* fails
+        // on the first attempt due to some lazy module loading issue in Safari...
+        // Therefore, we've to try it twice :-/
+        // see also: https://developer.apple.com/forums/thread/681201
+        await timeout(initStorage, 1000, timeoutError);
```

Comment:
I believe you need to call the function to get a promise, otherwise it always runs into the timeout.
```suggestion
        await timeout(initStorage(), 1000, timeoutError);
```

Reply:
Hm, then I was unfortunately right the first time, the patch doesn't work like this. The execution then never reaches the next line (as far as I can tell). Then, it just works because of the 1s delay... 🤔